Mathlib/RingTheory/SimpleModule/WedderburnArtin.lean

```diff
@@ -0,0 +1,87 @@
+/-
+Copyright (c) 2025 Junyan Xu. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Junyan Xu
+-/
+import Mathlib.RingTheory.FiniteLength
+import Mathlib.RingTheory.SimpleModule.Isotypic
+import Mathlib.RingTheory.SimpleRing.Congr
+
+/-!
+# Wedderburn-Artin Theorem
```

Comment:
In fact, please also include the statement in this module doc

Reply:
Same here, please see [d870b8b](https://github.com/leanprover-community/mathlib4/pull/24192/commits/d870b8b697d1c75dd77f9b6722e3d26aab190d3a)